client_test/crash_test.go

```diff
@@ -53,13 +59,14 @@ func TestCrashTolerance(t *testing.T) {
 		clientA.Close()
 		anotherMessageserviceA := messageservice.NewTestMessageService(alice.Address(), broker, 0)
 		anotherChainA, err := chainservice.NewSimulatedBackendChainService(sim, bindings, ethAccounts[0], logDestination)
+		anotherStoreA := store.NewPersistStore(alice.PrivateKey, dataFolder)
```

Comment:
Funky idea but I wonder about mixing the persist store and the mem store in a test like this?

Reply:
Wouldn't all the data in the `memStore` get lost, so the persist store would be starting fresh?